### PR TITLE
Redis cluster

### PR DESCRIPTION
**Pull request by otherview**
Initial push for the redis package.

Currently its failing because of race conditions with the mock package

### CODE REVIEW

**Comment by SkinyMonkey** [comment]
Would love to see this merged!

File: README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ But be aware that the API might change over time until 1.0.
 ## Sourcecode - Go
 This is the Go repository for Proto Actor.
 
-The C# implementation can be found here [https://github.com/AsynkronIT/protoactor-dotnet](https://github.com/AsynkronIT/protoactor-dotnet)
+The C# implementation can be found here [https://github.com/otherview/protoactor-dotnet](https://github.com/otherview/protoactor-dotnet)
```
Comment (alexeyzimarev) [inline README.md:20]: I am quite confused about all links changed to your private repos. What does it mean? I don't think changed in markdown files belong to the Redis cluster implementation at all.

Comment (otherview) [reply README.md:20]: Bad PR 🗡️ 

I need to test some things so I changed the repo. And I guess I forgot to put it back.
Apologies for that !

Comment (ccamel) [reply README.md:20]: It seems that a global replace of the project package name has been done during the fork (
0beb1f1), unfortunately.

**Comment by rogeralsing** [comment]
@otherview any chance this can be updated?

**Comment by otherview** [comment]
Hi @SkinyMonkey @alexeyzimarev  @rogeralsing !

I've got a bit of time to work on this and created https://github.com/AsynkronIT/protoactor-go/pull/393 :)

Closing this PR now :)

**Comment by SkinyMonkey** [comment]
@otherview Nice! Thank you for checking this :)

**Pull request closed without merging**